packages/css-library/src/stylesheets/base/va.scss

```diff
@@ -11,7 +11,7 @@ body {
 }
 
 html {
-  font-size: $em-base;
+  font-size: 100%; // Needs to be 100% to be able to grow and shrink with browser settings
```

Comment:
This [was changed to 16px](https://github.com/department-of-veterans-affairs/component-library/pull/1185/files) because of root font size updates related to USWDS v3 (it previously being 10px). Was this tested using Verdaccio to make sure everything still looks correct by default? 

![Screenshot 2024-12-26 at 8 59 08 AM](https://github.com/user-attachments/assets/427b2de2-4ae5-49ec-823e-d3dbeb54536f)

Reply:
I agree here with Jami. If this hasn't been tested with Verdaccio, then it needs to be. The entire typography change initiative we completed last summer was centered around this.